entity-service/src/main/kotlin/com/egm/stellio/entity/util/GraphBuilder.kt

```diff
@@ -0,0 +1,77 @@
+package com.egm.stellio.entity.util
+
+import com.egm.stellio.entity.service.EntityService
+import com.egm.stellio.entity.web.BatchEntityError
+import com.egm.stellio.shared.model.BadRequestDataException
+import com.egm.stellio.shared.model.ExpandedEntity
+import com.google.common.graph.Graph
```

Comment:
I see that the `Graph` API is "marked unstable with @Beta". Is there a roadmap to make it stable?

Reply:
From what I've read this is very unclear https://github.com/google/guava/issues/3421
I can swap to https://jgrapht.org/

Reply:
Well, not now at least.

Reply:
I've commited a version in which I migrated from guava to JgraphT (really fast as I only use basic stuff)